client/src/Login.js

```diff
@@ -50,12 +93,37 @@ function Login() {
                 item
                 xs={6}
                 style={{backgroundColor: '#444449', minWidth: '90%', minHeight: '90%', margin: 'auto', alignItems:"center"}}>
-                <p className={classes.mainContentStyle}>Login</p>
+                <p className={classes.mainContentStyle}>{buttonText}</p>
                 <p className={classes.SubHeadings} style={{marginBottom:'50px'}}>Welcome to CloudHaven</p>
                 <Grid container justify="center" alignItems="center">
-                  <TextField color="whitesmoke" className="textfield" label='username' variant="filled"/>
-                  <TextField color="whitesmoke" className="textfield" label='password' variant="filled"/>
-                  <Button variant="contained" color="primary" href="" className={classes.Button}>Login</Button> 
+                  {/* <TextField color="secondary" className="textfield" label='Username' variant="filled"/> */}
+                  <CssTextField
```

Comment:
These text boxes look great, but they are distinctly different than textboxes in other parts of our program. I had only used the material UI textboxes here because we use a lot of material UI components throughout the rest of our project. I could be easily convinced to use the CssTextfields I just figure we should be consistent. 
![DifferentTextbox](https://user-images.githubusercontent.com/38015797/108887985-c2a92380-75bf-11eb-902b-500a9b3cfe45.JPG)
![Overlap](https://user-images.githubusercontent.com/38015797/108887989-c341ba00-75bf-11eb-8af1-d9fbb116058b.JPG)